test/metabase/api/search_test.clj

```diff
@@ -370,6 +371,20 @@
       (is (= dashboard-count-results
              (set (unsorted-search-request-data :rasta :q "dashboard-count")))))))
 
+(deftest moderated-status-test
+  (let [search-term "moderated-status-test"]
+    (mt/with-temp [:model/Card {card-id :id} {:name "moderated-status-test"}]
+      (doseq [status ["verified" nil "verified"]]
+        (moderation-review/create-review! {:moderated_item_id card-id
+                                           :moderated_item_type "card"
+                                           :moderator_id         (mt/user->id :crowberto)
+                                           :status                status}))
+      (is (= [[card-id "card" "verified"]]
+             (->> (mt/user-http-request :crowberto :get 200 "search"
+                                        :q search-term)
+                  :data
+                  (map (juxt :id :model :moderated_status))))))))
```

Comment:
This is what `=?` or `partial=` is designed for

Reply:
Oh I see that you're also testing that the data contains one item. Never mind then